[SAASINT-3281][saas-ints][ccm] Update tags for saas cost integrations (#26078)
DataDog/documentation: content/en/cloud_cost_management/saas_costs.md

{{< tabs >}}
{{% tab "Databricks" %}}

> 1. Navigate to the [Databricks integration tile][101] in Datadog and click the **Configure** tab.

Reviewer: ```suggestion
1. Navigate to the [Databricks integration tile][101] in Datadog and click **Configure**.
```

2. Enter the workspace name, url, and access token corresponding to your Databricks account.
3. Under the **Select products to set up integration** section, click the toggle for each account to enable Databricks `Cloud Cost Management`.
4. Enter a `System Tables SQL Warehouse ID` corresponding to your Databricks instance's warehouse to query for system table billing data.

> 4. Click **Save Databricks Workspace**.

Reviewer: ```suggestion
5. Click **Save Databricks Workspace**.
```

| `billing_origin_product` | Product or feature originating the billing event (for example, JOBS, CLUSTERS). |
| `usage_type` | Type of usage being billed (for example, COMPUTE_TIME). |
| `job_run_id` | Identifier for the specific job run (if applicable). |
| `node_type` | Type of node used in this billing record (for example, m5d.large). |
| `destination_region` | Region where the workload is directed (if applicable). |
| `central_clean_room_id` | ID of the central clean room associated with the workload (if applicable). |
| `notebook_path` | Path to the notebook in Databricks (if applicable). |
| `job_name` | Name of the job in Databricks (if applicable). |
| `notebook_id` | ID of the notebook used in this billing record (if applicable). |
| `dlt_update_id` | Delta Live Table update ID associated with this usage (if applicable). |
| `job_id` | Unique identifier for the job in Databricks. |
| `dlt_maintenance_id` | Maintenance ID for Delta Live Tables (if applicable). |
| `run_name` | Name of the current job or workflow run (if applicable). |
| `instance_pool_id` | ID of the instance pool used (if applicable). |
| `cluster_id` | ID of the cluster associated with this usage. |

> | `endpoint_id` | ID of the endpoint, for SQL-based or serving-related usage (if applicable). |

Reviewer: ```suggestion
| `endpoint_id` | ID of the endpoint for SQL-based or serving-related usage (if applicable). |
```

| `account_locator` | Locator for the account where the usage was consumed. |
| `region` | Name of the region where the account is located. |
| `service_level` | Service level (edition) of the Snowflake account (Standard, Enterprise, or Business Critical). |
| `user_name` | Name of the user or service account associated with the query. |
| `warehouse_id` | Identifier for the warehouse generating the cost. |
| `warehouse_name` | Name of the warehouse associated with this usage. |
| `warehouse_size` | Size of the warehouse (for example, Large, Medium). |

> | `cost_type` | Type of cost associated with the usage. Possible values include:<br> - `CLOUD_SERVICES`: General costs related to Snowflake's underlying cloud services, and not associated with warehouse usage.<br> - `IDLE_OR_LESS_100MS`: Costs from warehouse idle time or queries that completed in under 100 milliseconds. Unattributed to specific queries. Falls under the **warehouse_metering** service type.<br> - `QUERY_ATTRIBUTION`: Costs attributed to specific queries, grouped by the parameterized query hash. For these costs, the parameterized query associated with this cost can be found under **charge description**. Falls under the **warehouse_metering** service type. |

Reviewer: ```suggestion
| `cost_type` | Type of cost associated with the usage. Possible values include:<br> - `CLOUD_SERVICES`: General costs related to Snowflake's underlying cloud services, excluding warehouse usage.<br> - `IDLE_OR_LESS_100MS`: Costs from warehouse idle time or queries that completed in under 100 milliseconds. Unattributed to specific queries. Falls under the **warehouse_metering** service type.<br> - `QUERY_ATTRIBUTION`: Costs attributed to specific queries, grouped by the parameterized query hash. For these costs, the parameterized query associated with this cost can be found under **charge description**. Falls under the **warehouse_metering** service type. |
```

| `query_hash` | Unique hash representing a parameterized version of the query for attribution purposes. Only found for **query attribution** costs. |
| `query_hash_version` | Version of the Snowflake query hash algorithm used to generate `query_hash`. Only found for **query attribution** costs. |
| `database_name` | Name of the database in which the query was executed (if applicable). Only found for **query attribution** costs. |
| `balance_source` | Source of the funds used to pay for the daily usage. The source can be one of the following:<br>- **capacity**: Usage paid with credits remaining on an organization’s capacity commitment.<br>- **rollover**: Usage paid with rollover credits. When an organization renews a capacity commitment, unused credits are added to the balance of the new contract as rollover credits.<br>- **free usage**: Usage covered by the free credits provided to the organization.<br>- **overage**: Usage that was paid at on-demand pricing, which occurs when an organization has exhausted its capacity, rollover, and free credits.<br>- **rebate**: Usage covered by the credits awarded to the organization when it shared data with another organization. |

> | `service_type` | Type of usage. Possible service types include:<br>- **automatic_clustering**: Refer to Automatic Clustering.<br>- **data_transfer**: Refer to Understanding data transfer cost.<br>- **logging**: Refer to Logging and Tracing Overview.<br>- **materialized_view**: Refer to Working with Materialized Views.<br>- **replication**: Refer to Introduction to replication and failover across multiple accounts.<br>- **query_acceleration**: Refer to Using the Query Acceleration Service.<br>- **search_optimization**: Refer to Search Optimization Service.<br>- **serverless_task**: Refer to Introduction to tasks.<br>- **snowpipe**: Refer to Snowpipe.<br>- **snowpipe_streaming**: Refer to Snowpipe Streaming.<br>- **storage**: Refer to Understanding storage cost.<br>- **warehouse_metering**: Refer to Virtual warehouse credit usage. Does not indicate usage of serverless or cloud services compute. |

Reviewer: Quick question. When you say "refer", are you talking about a specific section in the docs, or another query parameter?

Author: I think these (outside of warehouse metering) reference specific Snowflake products that have cost models outside of the main compute-based one.

| Tag Name | Tag Description |
|---|---|
| `credit_coupon_code` | Code of any coupon or credit applied to this cost entry (if available). |

> | `product_name` | Name of the specific product being billed (e.g., "North America Bandwidth"). |

Reviewer: ```suggestion
| `product_name` | Name of the specific product being billed (for example, "North America Bandwidth"). |
```

| `product_group` | Group or category of the product, such as "Full Site Delivery". |
| `product_line` | Line of products to which this item belongs, for example, "Network Services". |

> | `usage_type` | Type of usage being billed (e.g., "Bandwidth"). |

Reviewer: ```suggestion
| `usage_type` | Type of usage being billed (for example, "Bandwidth"). |
```

> | `region` | Region where the service usage occurred, such as "North America". |

Reviewer: ```suggestion
| `region` | Region where the service usage occurred (for example, "North America"). |
```
